docs/contributing/pull-requests/code-review.md

```diff
@@ -46,20 +55,36 @@ requirements, which may not necessarily be in line with the previous solution.
 
 Please use the review statuses appropriately.
 
+:::tip Think about your audience
+
+When providing comments or requesting changes, keep in mind the experience level of the author.
+
+If the engineer is new to the company and the codebase or less experienced overall, they would
+probably welcome more explicit background on your concerns and more concrete suggestions, whereas a
+more experienced engineer would prefer general guidance so that they can solve the problem
+themselves.
+
+:::
+
 #### Comment
 
 Comment is a great way to discuss things without explicitly approving or requesting changes.
 
+We use [conventional comments](https://conventionalcomments.org) to help convey the intent and
+expected action taken from each comment on the PR.
```

Comment:
I don't think this has been established, The template repository uses a variant with emojis but it's ambiguous. I prefer the more explicit nature of conventional commits since it defines what is blocked and what isn't.

Reply:
Deliberately ambiguous too. There were a lot of comments on https://github.com/bitwarden/template/pull/5 to get it to a place folks were liking. Conventional comments (and commits) can be a lot of work in practice and if anything it's an opt-in to how you want to interact. I'd caution us to not over-prescribe here.

Reply:
I've found the ambiguity to be a disservice. For example is 🎨 blocking or personal preference?

Reply:
A review can be a comment, approval, or changes requested, and once you push commits it resets. This is an author action that to me implies re-review, so it'd prompt that of changes requested -- blocking or not -- it's due for another look and that matter would be discussed; if it wasn't addressed by the author they consider it non-blocking. The question instead might be which side moves forward in an impasse, and that's addressed elsewhere I thought.

Reply:
I prefer conventional comments to the ambiguity of emoji. It's helpful to understand the position of a commenter when addressing comments and making your position clear can save a round trip.

The issue is that a _review_ can be comment, approve, or request changes, but a comment cannot be. It's extremely rare for all of my comments to be of only one type.

Reply:
```suggestion
We use [conventional comments](https://conventionalcomments.org) to help convey the intent and
expected action taken from each comment on the PR. The PR author can disagree and take different action, but clarifying your position while commenting helps make your intent clearer.
```

Reply:
I left the Comments section as-is, and I've broken out the changes into a separate branch to avoid having both conversations on this one.  I had assumed there was agreement on their use, so I'll use this opportunity to break apart into smaller PRs 😄 .